interface/patient_file/download_template.php

```diff
@@ -338,6 +373,16 @@ function doSubs($s)
 $hisrow = sqlQuery("SELECT * FROM history_data WHERE pid = ? " .
   "ORDER BY date DESC LIMIT 1", array($pid));
 
+$vitalsEncounter = $encounter + 1;
+//Get vitals from latest encounter
+$vitalsrow = sqlQuery("SELECT * FROM form_vitals AS FORM_VITALS " .
```

Comment:
wouldn't mess with casing changes (use lowercase as is in the table names) since this can cause unpredictable results across different operating systems